src/util/isAbbreviation.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * Function: isAbbrev.
+ *
+ * @param str1 The sentence just added into the result sentence, it contains the sentence and the splitter.
+ * @param s The current sentence which is right behind the spliter.
+ * @param spliter2 The second spliter.
+ * @returns A boolean value that tells whether the dot comes from an Abbrev word, and shouldn't be split
+ * Examples: Mr. Dr.q Apt. Prof. Ph.D.
+ */
+export function isAbbrev(str1 : string, s: string, spliter2 : string) {
+
+  return isAbbrEndSplitter(str1, s) || isAbbrMidSplitter(str1, s) || isAbbrDoubleSplitter(str1, s, spliter2)
+}
```

Comment:
Use a single default export:

```js
export default isAbbrev
```

Reply:
Oops! I actually got this wrong. You were right the first time :). No problem though.